include/oneapi/dpl/internal/serial_numeric_impl.h

```diff
@@ -18,9 +18,9 @@
 #define _ONEDPL_SERIAL_NUMERIC_IMPL_H
 
 #if (__cplusplus >= 201703L)
-#    include <functional>
-#    include <iterator>
-#    include <numeric>
+#    include <functional>       // KSATODO: we may include this file without __cplusplus check https://en.cppreference.com/w/cpp/utility/functional/function - since C++11
+#    include <iterator>         // KSATODO: we may include this file without __cplusplus check https://en.cppreference.com/w/cpp/header/iterator
+#    include <numeric>          // KSATODO: we may include this file without __cplusplus check https://en.cppreference.com/w/cpp/header/numeric
```

Comment:
Same thing as in `serial_algorithm_impl.h`

Reply:
Fixed: removed support of C++14 too